src/Components/Home/Home.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from 'react'
+import Book from '../Book/Book'
+import './Home.css'
+import { RxCaretRight, RxCaretLeft } from "react-icons/rx"
+import { useQuery, gql } from '@apollo/client'
+
+const BOOK_SEARCH = gql`
+  query BookSearch($title: String!) {   
+    bookSearch(title: $title) {
+        id
+        title
+        imageUrl
+      }
+  }`
+
+const BOOKS_DATA = gql `
+    query books {
+      books {
+      id 
+      imageUrl
+    }
+}`
+
+interface book {
```

Comment:
We should probably meet to discuss putting these into their own files!

Reply:
Yeah I think that's a good idea!